Topics/User_Experience/AB_Testing.md

```diff
@@ -0,0 +1,36 @@
+# A/B Testing
+
+## **Introduction:**
+A/B testing is a research method used to compare two versions of a web page or mobile app to determine which variation of the UI results in better performance. It allows individuals and companies to optimize user experience and make improvements to their application. 
```

Comment:
I think maybe talk about the alternatives to A/B testing would be helpful to know why it's so well-known and ubiquitous, or maybe talking about what goes wrong if A/B testing isn't done? Things like canary setups, private betas for users, etc. might be good to contrast against.

Reply:
I agree it would be helpful to include other alternatives